runtime/v1/shim/client/client.go

```diff
@@ -320,26 +316,5 @@ func (c *Client) signalShim(ctx context.Context, sig syscall.Signal) error {
 	if err := unix.Kill(pid, sig); err != nil && err != unix.ESRCH {
 		return err
 	}
-	// wait for shim to die after being signaled
-	select {
-	case <-ctx.Done():
-		return ctx.Err()
-	case <-c.waitForExit(pid):
-		return nil
-	}
-}
-
-func (c *Client) waitForExit(pid int) <-chan struct{} {
```

Comment:
could we just keep it and add log for warning after timeout? thanks

Reply:
done @fuweid